airflow/triggers/base.py

```diff
@@ -109,7 +109,7 @@ class TriggerEvent:
     events.
     """
 
-    def __init__(self, payload: Any):
+    def __init__(self, payload: Any = None):
```

Comment:
Since it isn’t particularly useful to have a null payload for non-ending events, maybe this should stay required, and only overridden in BaseTaskEndEvent?

Reply:
sure